tpm2tools/keys.go

```diff
@@ -273,7 +283,18 @@ func (k *Key) Unseal(in *tpmpb.SealedBytes, cOpt CertifyOpt) ([]byte, error) {
 		creationHash := sessionHashAlg.New()
 		creationHash.Write(in.GetCreationData())
 		if _, _, err = tpm2.CertifyCreation(k.rw, "", sealed, tpm2.HandleNull, nil, creationHash.Sum(nil), tpm2.SigScheme{}, ticket); err != nil {
```

Comment:
Is there a way to only trigger this fallback behavior on a specific type of error? If not that's ok. I would just like to only fallback if we hit the bug

Reply:
The err being returned is:

> parameter 3, error code 0x1a : the TPM was unable to unmarshal a value because there were not enough octets in the input buffer

We can try to use strings.contains() to extract the error code from the err string, but I don't think it is a good idea, since we can not get the error code directly from the err object.

Reply:
done